scripts/us_ca_chhs/healthcare_facility_bed_counts/preprocess_csv.py

```diff
@@ -0,0 +1,261 @@
+            'CALicensedHealthcareFacilityName': row_dict['FACNAME'],
+            'CALicensedHealthcareFacilityType': 'dcs:' + FDR_to_type[row_dict['FAC_FDR']],
+            'Count_HospitalBed_SkilledNursingBed': None,
+            'Count_HospitalBed_SpecialTreatmentProgramBed': None,
+            'Count_HospitalBed_IntermediateCareHabilitativeBed': None,
+            'Count_HospitalBed_IntermediateCareNursingBed': None,
+            'Count_HospitalBed_CongregateLivingHealthFacilityBed': None,
+            'Count_HospitalBed_IntermediateCareBed': None,
+            'Count_HospitalBed_CoronaryCareBed': None,
+            'Count_HospitalBed_IntensiveCareBed': None,
+            'Count_HospitalBed_IntensiveCareNewbornNurseryBed': None,
+            'Count_HospitalBed_PerinatalBed': None,
+            'Count_HospitalBed_UnspecifiedGeneralAcuteCareBed': None,
+            'Count_HospitalBed_BurnBed': None,
+            'Count_HospitalBed_PediatricBed': None,
+            'Count_HospitalBed_RenalTransplantBed': None,
+            'Count_HospitalBed_RehabilitationBed': None,
+            'Count_HospitalBed_AcuteRespiratoryCareBed': None,
+            'Count_HospitalBed_AcutePsychiatricCareBed': None,
+            'Count_HospitalBed_ChemicalDependencyRecoveryBed': None,
+            'Count_HospitalBed_PediatricIntensiveCareUnitBed': None,
+            'Count_HospitalBed_LaborAndDeliveryBed': None,
+            'Count_HospitalBed_IntermediateCareDevelopmentallyDisabledBed': None,
+            'Count_HospitalBed_PsychiatricHealthBed': None,
+            'Count_HospitalBed_PediatricDayAndRespiteCareBed': None,
+            'Count_HospitalBed_DialysisStationsBed': None,
+            'Count_HospitalBed_CorrectionalTreatmentCenterBed': None
+        }
+        new_row[stat_var] = row_dict['BED_CAPACITY']
+        new_df = new_df.append(new_row, ignore_index=True)
+
+new_df.to_csv('CA_Licensed_Healthcare_Facility_Types_And_Counts.csv')
+
+# Output as a tmcf file.
+# Facility node, facility containedIn a county
+TEMPLATE_MCF_FAC_1 = """
+Node: E:CA_Licensed_Healthcare_Facility_Types_And_Counts->E0
+healthcareFacilityName: C:CA_Licensed_Healthcare_Facility_Types_And_Counts->CALicensedHealthcareFacilityName
+typeOf: schema:Hospital
+dcid: C:CA_Licensed_Healthcare_Facility_Types_And_Counts->CALicensedHealthcareFacilityBedFACID
+healthcareFacilityType: C:CA_Licensed_Healthcare_Facility_Types_And_Counts->CALicensedHealthcareFacilityType
+containedIn: C:CA_Licensed_Healthcare_Facility_Types_And_Counts->GeoId
+"""
+
+# observation node, observationAbout facility
+TEMPLATE_MCF_TEMPLATE_1 = """
+Node: E:CA_Licensed_Healthcare_Facility_Types_And_Counts->E{index}
+typeOf: StatVarObservation
+variableMeasured: {stat_var}
+observationDate: C:CA_Licensed_Healthcare_Facility_Types_And_Counts->Date
+observationAbout: E:CA_Licensed_Healthcare_Facility_Types_And_Counts->E0
+value: C:CA_Licensed_Healthcare_Facility_Types_And_Counts->{stat_var}
+"""
```

Comment:
Also move these declarations towards the top (this will help us when we start to modularize the code)